src/common/createSlider.jsx

```diff
@@ -120,6 +120,22 @@ export default function createSlider(Component) {
       utils.pauseEvent(e);
     }
 
+    onFocus = (e) => {
+      const isVertical = this.props.vertical;
+
+      if (utils.isEventFromHandle(e, this.handlesRefs)) {
+        const handlePosition = utils.getHandleCenterPosition(isVertical, e.target);
+
+        this.dragOffset = 0;
```

Comment:
I think we can keep it, since  `this.dragOffset = 0;` is to restore drag status.